icaruscode/Decode/DecoderTools/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ art_make( SUBDIRS details
           LIB_LIBRARIES
                         icaruscode_Utilities
                         sbnobj_Common_PMT_Data
+                        canvas
```

Comment:
This is from "Refurbished PMT readout configuration discovery code" 

would it work without actual file change?

Reply:
Oh, I see. I don't think this is going to be an issue for anything.